components/ministry-teams/MinistryTeamsCard.jsx

```diff
@@ -29,6 +41,71 @@ export default function MinistryTeamsCard(props) {
   }
   currMT.leadersNamesString = stringifyLeaderNames(currMT.leadersNames);
 
+  const removalText = () => {
+    return ("Are you sure you would like to permanently remove this Ministry Team " +

```

Comment:
Same as my comment for community group. I would say it should probably include the Ministry Team name, but that isn't critical.